### PR TITLE
Possible Fix Custom Video Controls

### PR DESCRIPTION
**Pull request by grafixeyehero**
Tested 
Firefox: 73.0b6 (64-bit)
Chrome: 79.0.3945.117
Microsoft Edge 44.18362.449.0

### CODE REVIEW

**Comment by MrTimscampi** [comment]
I can confirm this works on Firefox 72.0.1 (64bit) and Chrome 79.0.3945.130 (64bit).

File: src/components/htmlvideoplayer/plugin.js [hunk 1]
```diff
@@ -1378,6 +1378,10 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
             }
         }
 
+        function enableCustomVideoControls() {
+            return true;
+        }
```
Comment (JustAMan) [inline src/components/htmlvideoplayer/plugin.js]: heh, I want some explanation here... you're basically always default to `else` clause, which (as I judge from comments) could break on WebOS which have old-ish Chrome bundled.

Comment (dkanada) [reply src/components/htmlvideoplayer/plugin.js]: Seems like this might be resolved by simply removing the CSS rule.

Comment (JustAMan) [reply src/components/htmlvideoplayer/plugin.js]: Those things there were for a reason, so I'm sensing that we'd need them for some edge cases...

Comment (dkanada) [reply src/components/htmlvideoplayer/plugin.js]: I specifically mean the four lines of CSS he removed, which look like they would globally hide all custom video controls on firefox.

Comment (grafixeyehero) [reply src/components/htmlvideoplayer/plugin.js]: > 
> 
> heh, I want some explanation here... you're basically always default to `else` clause, which (as I judge from comments) could break on WebOS which have old-ish Chrome bundled.

i am limited here

**Comment by dkanada** [comment]
resolved in #726 for now

**Pull request closed without merging**